SmartCharger.Business/Services/EventService.cs

```diff
@@ -33,8 +33,8 @@ public async Task<EventResponseDTO> GetUsersChargingHistory(int userId, int page
                 {
                     string searchLower = search.ToLower();
 
-                    query = query.Where(e => e.Card.Name.ToLower().Contains(searchLower)
-                                           || e.Charger.Name.ToLower().Contains(searchLower));
+                    query = query.Where(e => e.Card.Name.ToLower().Contains(search)
```

Comment:
I think this change doesn't make sense, because this way search is case sensitive.

Reply:
Yes you are right. Now I noticed this "dumb" mistake. I will fix this, but also I will fix this in other services, because this mistake is repeating in other services too. For that I will make new hotfix branch